config/routes.rb

```diff
@@ -4,13 +4,17 @@
   root 'static_pages#home'
   get 'myscope' => 'scope#myscope'
   get 'help'    => 'static_pages#help'
+
   resources :languages
 
   resources :countries
 
   resources :percentages
 
-  resources :schools, only: [:index, :edit, :show]
+  resources :schools, only: [:index, :show]
+  get 'offers'   => 'schools#edit'
+  put 'offers'   => 'schools#update'
+  patch 'offers' => 'schools#update'
```

Comment:
That's a very interesting point and I actually wanted to discuss this with you.
If you notice, devise generates routes such as 
edit_school_registration GET    /schools/edit(.:format)          devise/registrations#edit
whereas the usual route for this would be
/schools/:id/edit
Since the offers are related with schools I wanted to be consistent and do it the devise way. Actually
get 'schools/offers'   => 'schools#edit'
would be much better.
Let me know your opinion about this